CP01CNSM/CP01CNSM_D00013_ingest.csv

```diff
@@ -0,0 +1,35 @@
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl27/syslog/*syslog.log,CP01CNSM-RID27-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl27/optaa1_decimated/*optaa1.log,CP01CNSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl27/flort/*flort.log,CP01CNSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl27/ctdbp1/*ctdbp1.log,CP01CNSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl27/dosta1/*dosta1.log,CP01CNSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/irid2shore/*cpm_status*.txt,CP01CNSM-SBC11-00-CPMENG000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl11/syslog/*syslog.log,CP01CNSM-SBD11-00-DCLENG000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl11/mopak/*mopak.log,CP01CNSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl11/hyd1/*hyd1.log,CP01CNSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl11/metbk1/*metbk1.log,CP01CNSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/syslog/*syslog.log,CP01CNSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/hyd2/*hyd2.log,CP01CNSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/pco2a/*pco2a.log,CP01CNSM-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/wavss/*wavss.log,CP01CNSM-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/metbk2/*metbk2.log,CP01CNSM-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00013/cg_data/dcl12/fdchp/*fdchp.log,CP01CNSM-SBD12-08-FDCHPA000,telemetered,Available,
```

Comment:
Looks good!

Reply:
Looks good 
CP01CNSM-MFD35-06-PHSEND000 is pointing to cg_data/dcl37 instead of cg_data/dcl36 because it was moved to a different DCL bottle due to port issues.